### ISSUE TITLE
Cannot read property 'getBoundingClientRect' of null

### ISSUE THREAD

**Issue by bionetmarco** [smooch/smooch-js#400]
Certain times, the Smooch window is showing below the page, and inspecting in Chrome pushes the chat window up to become visible. Unfortunately I'm not entirely sure how to reproduce the issue.

**Comment by alexdorsch** [comment]
This can be reproduced by sending a whisper message to a client that has no previous messages in the conversation component.

```
Uncaught (in promise) TypeError: Cannot read property 'getBoundingClientRect' of undefined(…)
    t.ConversationComponent.t.componentWillUpdate	@ conversation.jsx:150
```

**Comment by lemieux** [comment]
@alexdorsch what kind of whisper are you using? On app start or on an event?

**Comment by alexdorsch** [comment]
@lemieux we are using a custom event to trigger the whisper

**Comment by lemieux** [comment]
Hey @bionetmarco & @alexdorsch, I just want to let you know that this bug was fixed with the 3.5.1 release.
